models/parent.ts

```diff
@@ -0,0 +1,14 @@
+import type { provinces, locales } from "@prisma/client";
+
+export type CreateParentInput = {
```

Comment:
I like the type name!

Reply:
I think we can use this object to also return from the GET request in which case I think just `Parent` would be a better name. cc @feng-eric

Reply:
I think it might be better to have a type for returning from the GET request too. Take for example if we have `include` in the query. Then, the fields in the record returned from `prisma` wouldn't match this as it would include the relational fields